transaction.proto

```diff
@@ -15,12 +16,14 @@ message Fee {
     JubJubCompressed R = 3;
     JubJubCompressed pk_r = 4;
 }
+
 message Note {
-    JubJubCompressed randomness = 1;
-    JubJubCompressed pk_r = 2;
-    JubJubCompressed commitment = 3;
-    BlsScalar nonce = 4;
-    PoseidonCipher encypted_data = 5;
+    int32 note_type = 1;
+    JubJubCompressed value_commitment = 2;
+    JubJubScalar nonce = 3;
+    StealthAddress pk_r = 4;
```

Comment:
as already mentioned in a previous review, `StealthAddress` and `pk_r` are two different thing: `pk_r` is a part of `StealthAddress`. So the name would be `stealth_address`, or `addr` or whatever you like, but not `pk_r` since would be misleading (you will end up with `note.pk_r.pk_r` structure)

Reply:
Yep, gotcha